src/map.tsx

```diff
@@ -273,14 +275,15 @@ const ReactMapboxFactory = ({
       };
 
       const map = new MapboxGl.Map(opts);
+      this.setState({ ...this.state, map });
```

Comment:
You don't need to spread `this.state`, `setState` is doing a merge and not a replace